src/styles/blocks/_cards.scss

```diff
@@ -0,0 +1,135 @@
+.container {
+  max-width: 1100px;
```

Comment:
checklist 11th item, you should do the calculation for it
[LAYOUT] - don't hardcode max-width, your catalog container should work fine for any number of cards

Reply:
Done.